netam/pretrained.py

```diff
@@ -43,6 +46,13 @@
         MODEL_TO_LOCAL[model] = (local_file, models_dir)
 
 
+PRETRAINED_MULTIHIT_MODELS = {
+    # Trained using the notebook
+    # thrifty-experiments-1/human/multihit_model_exploration.ipynb
```

Comment:
Perhaps just explain a little 

* these model names are arbitrary
* but we use the model name to express where it came from